MediaGalleryUi/Model/GetImageDetailsByAssetId.php

```diff
@@ -109,6 +119,10 @@ public function execute(int $assetId): array
             'title' => $asset->getTitle(),
             'image_id' => $assetId,
             'details' => [
+                [
+                    'title' => __('Type'),
+                    'value' => $this->getTypeImageByMediaTypeId($mediaTypeId),
```

Comment:
Please retrieve the type from the contenty_type field of the media_gallery_asset.

If the asset comes from Adobe Stock (there is information stored in adobe_stock_asset) - please update this information in `\Magento\AdobeStockImageAdminUi\Plugin\AddAdobeStockImageDetailsPlugin`